File: flask_api/create_env.py
# Create a new file called create_env.py in your flask_api folder
# Then run this script

content = """OPENAI_API_KEY=your_new_api_key
DB_HOST=localhost
DB_USER=root
DB_PASSWORD=
DB_NAME=epm_database
GOOGLE_MAPS_API_KEY=your_google_maps_key"""

with open('.env', 'w', encoding='utf-8') as f:
    f.write(content)

File: create_env.php
<?php
$content = "OPENAI_API_KEY=your-api-key-goes-here
DB_HOST=localhost
DB_USER=root
DB_PASSWORD=
DB_NAME=epm_database";

file_put_contents(__DIR__ . '/.env', $content);
echo "Created .env file in: " . __DIR__;
?> 